src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -25,7 +25,6 @@ __PORTABLETARGETS_PROJECT_CONTENT="
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include=\"MicroBuild.Core\" Version=\"$__MICROBUILD_VERSION\" />
-    <PackageReference Include=\"Microsoft.Portable.Targets\" Version=\"$__PORTABLETARGETS_VERSION\" />
```

Comment:
Same here, I believe they get copied somewhere so we should remove that piece of code as well